scadalts-ui/src/components/watch_list/components/ChartSettingsStaticComponent.vue

```diff
@@ -124,48 +65,45 @@ export default {
 			let loadedData = JSON.parse(localStorage.getItem(`MWL_${this.watchListName}_P`));
 			if (!!loadedData) {
 				if (loadedData.type === this.CHART_TYPE) {
-					this.startDate = new Date(loadedData.startDate);
-					this.startTime = new Date(loadedData.startTime);
-					this.endDate = new Date(loadedData.endDate);
-					this.endTime = new Date(loadedData.endTime);
+					this.startDate = loadedData.startDate;
+					this.startTime = loadedData.startTime;
+					this.endDate = loadedData.endDate;
+					this.endTime = loadedData.endTime;
 				}
+			} else {
+				let today = new Date();
+				this.startDate = today.toISOString().slice(0, 10);
+				this.endDate = today.toISOString().slice(0, 10);
+				let tempTime = today.toLocaleTimeString().split(":")
+				this.startTime = `${Number(tempTime[0])-1}:${tempTime[1]}`;
```

Comment:
today.toLocaleTimeString() may return 12 hours am/pm format
"3:40:09 PM"
If we drop the last term the value becomes ambiguous.
And -1 hour has the issue 1 hour before could be the previous day
Javascript Date implementation issues/bugs.
I think we should decide between importing dayjs or Temporal polyfil

https://github.com/tc39/proposal-temporal
https://github.com/iamkun/dayjs

I re-wrote the whole block according I understood this code is intended to do:

const now = dayjs()
const oneHourBefore = now.subtract(1, 'hour')
this.startDate = oneHourBefore.format("YYYY-MM-DD")
this.endDate = now.format("YYYY-MM-DD")
this.startTime = oneHourBefore.format("H:M")
this.endTime = now.format("H:M")

It's usually not a good idea to persist dates in local format instead of persisting in ISO or UTC and formating just before displaying.